tests/metrics/genai/test_genai_metrics.py

```diff
@@ -1130,3 +1132,66 @@ def test_make_custom_prompt_genai_metric_validates_input_kwargs():
         custom_judge_prompt_metric.eval_fn(
             input=pd.Series(inputs),
         )
+
+
+def test_log_custom_metric_config_in_make_genai_metric_from_prompt():
+    custom_judge_prompt = "This is a custom judge prompt that uses {input} and {output}"
+
+    custom_metric = make_genai_metric_from_prompt(
+        name="custom",
+        judge_prompt=custom_judge_prompt,
+        greater_is_better=False,
+        parameters={"temperature": 0.0},
+    )
+
+    assert custom_metric.custom_metric_config is not None
```

Comment:
I'd remove this. The next assert statement validates that `custom_metric_config` is not None.

Reply:
okie!